compiler/fir/checkers/checkers-component-generator/src/org/jetbrains/kotlin/fir/checkers/generator/diagnostics/FirDiagnosticsList.kt

```diff
@@ -166,6 +166,8 @@ object DIAGNOSTICS_LIST : DiagnosticList() {
         val INAPPLICABLE_LATEINIT_MODIFIER by error<FirSourceElement, KtModifierListOwner>(PositioningStrategy.LATEINIT_MODIFIER) {
             parameter<String>("reason")
         }
+
+        val VARARG_OUTSIDE_PARENTHESES by error<FirSourceElement, PsiElement>()
```

Comment:
s/PsiElement/KtExpression/g ?

Reply:
I forgot to answer this: https://github.com/JetBrains/kotlin/pull/4143#discussion_r585285315 So, please double-check on what type the counterpart of old FE's diagnostic is reported.

Reply:
Thanks for the explanation!